### PR TITLE
mes-6188 add driver number to delegated examiner rekey search results

### PR DESCRIPTION
**Pull request by piercemorris**
## Description

Little add to show a driver's number on the delegated examiner rekey search results. This is so that a rekey user can verify they are on the right test.

## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [x] PR link added to JIRA ticket
- [x] One review from each scrum team
- [x] Squashed commit contains the JIRA ticket number

## Screenshots (optional)

NB vehicle details now removed
Simulator Screen Shot - iPad Pro (10 5-inch) - 2020-11-25 at 18 36 53

#### Journal page not affected by the changes made to the common component

Simulator Screen Shot - iPad Pro (10 5-inch) - 2020-11-26 at 09 59 10

### CODE REVIEW

File: src/components/test-slot/test-slot/test-slot.html [hunk 1]
```diff
@@ -66,6 +66,8 @@
           </progressive-access>
         </ion-col>
         <ion-col class="vehicle-details-col" no-padding>
+          <h3 [style.padding]="'5px 10px'" *ngIf="delegatedTest && slot.booking.candidate.driverNumber">{{slot.booking.candidate.driverNumber}}</h3>
```
Comment (szabikr) [inline src/components/test-slot/test-slot/test-slot.html]: Create a css class or id and put the stylings in the SASS file, pls

**Comment by szabikr** [comment]
For some reason the date is moved to the top. It should be underneath the Indicator/time. See Miro boards for reference

**Review by szabikr — APPROVED** [review]
Looks good now!

Just a side note, would be curious why does the vehicle details is displayed when we don't have any actual data to show?

**Comment by piercemorris** [comment]
> Looks good now!
> 
> Just a side note, would be curious why does the vehicle details is displayed when we don't have any actual data to show?

Confirmed that they do not need to be there and have been removed

**Review by tomcrawleyy — APPROVED** [review]

**Pull request merged**